site/data/charts/docker.json

```diff
@@ -1,89 +1,90 @@
 {
   "title": "Docker Pulls",
   "description": "Docker pulls since the ZAP Docker images were published.",
-  "columns": ["Version" , "live", "stable", "weekly", "bare", "zap-bare", "zap-nightly", "zap-stable", "zap-weekly"],
+  "columns": ["Version" , "live", "stable", "weekly", "zap2docker-bare", "zap2docker-live", "zap2docker-stable", "zap2docker-weekly", "zap-bare", "zap-nightly", "zap-stable", "zap-weekly"],
```

Comment:
Should these ones be back?

Reply:
Probably yes - I'll look into it, and the jump in ascans...

Reply:
This should fix that problem: https://github.com/zapbot/zap-mgmt-scripts/pull/188
Looks good locally but I still need to sanity check the ascans..